inference-engine/src/vpu/graph_transformer/src/middleend/pass_manager.cpp

```diff
@@ -250,7 +240,16 @@ PassSet::Ptr PassManager::buildMiddleEnd() {
 
     ADD_PASS(swConvAdaptation);
     ADD_PASS(swDeconvAdaptation);
+
+    //
+    // Replace Global AvgPooling with ReduceMean
+    //
+    if (env.config.enableReplaceWithReduceMean) {
+        ADD_PASS(replaceWithReduceMean);
+        ADD_DUMP_PASS("replaceWithReduceMean");
+    }
```

Comment:
If you need to move passes after some anothers, please document it at least in comment

Reply:
Added comment